icu4j/main/classes/core/src/com/ibm/icu/text/ConstrainedFieldPosition.java

```diff
@@ -344,6 +346,11 @@ public boolean matchesField(Field field, Object fieldValue) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     * @draft ICU 64
+     * @provisional This API might change or be removed in a future release.
+     */
```

Comment:
Even `@Override`, we still need API status tag. In this case, the API status should match the class's status.

Reply:
I am always confused about API status of overrides. I thought we had said recently that they should match the status of what they inherit, but when the whole class is new using the same status as the class makes sense... so ok.